packages/genesys-spark/src/style/_mixins.scss

```diff
@@ -49,3 +49,16 @@
     var(--gse-ui-menu-boxShadow-blur) var(--gse-ui-menu-boxShadow-spread)
     var(--gse-ui-menu-boxShadow-color);
 }
+
+@mixin gux-toolbar-prevent-initial-flicker-workaround {
+  opacity: 0;
+  animation-name: show-toolbar;
+  animation-delay: 0.5s;
+  animation-fill-mode: forwards;
+
+  @keyframes show-toolbar {
+    to {
+      opacity: 1;
+    }
+  }
+}
```

Comment:
Just for my own learning, if this is only used in one place why is it a mixin?

Reply:
@daragh-king-genesys  suggested above to make it mixin so that future devs will be able to know what it is for because at first glance you probably would be wondering why there is a delay on showing the toolbar.

Reply:
You can reproduce this glitchy initial render if you go to the live docs and go to the condensed layout or even the icon only mode for the toolbar. When you refresh the page you will see a flicker for a split second while the component is trying to calculate the correct layout to use.

Reply:
Ah ok, how would we qualify what gets included in general mixins vs what is a mixin declared in a component scss file? 

Might be worth adding a comment with a ref to the ticket somewhere just since it's not that intuitive, like you were saying.

Reply:
Actually good spot I am not sure why I put it in the general mixins :) I will just define in in the component file